### PR TITLE
Newcarddeck

### PR DESCRIPTION
**Pull request by brucekirk735**

### CODE REVIEW

**Comment by jefflaub12** [comment]
Also looks like you have an extra </div> there  
</div>
      {{/each}}
  </div>j

File: views/shop/pricing.hbs [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
           <div class="price pull-left">${{ this.price }}</div>
           <a href="/add-to-cart/{{this._id}}" class="btn btn-success pull-right" role="button">Add to Shopping Cart</a>
         </div>
+        <p class="card-text"><small class="text-muted"></small></p>
       </div>
     </div>
-  </div>
-    {{/each}}
+      {{/each}}
+  </div>j
```
Comment (jefflaub12) [inline views/shop/pricing.hbs:23]: What is j for?

File: views/shop/pricing.hbs [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+       </div>
+       <div class="card-body">
+        <h5 class="card-title text-center">{{ this.title }}</h5>
+        <p class="card-text">{{ this.description }}</p>
         <div class="clearfix">
           <div class="price pull-left">${{ this.price }}</div>
           <a href="/add-to-cart/{{this._id}}" class="btn btn-success pull-right" role="button">Add to Shopping Cart</a>
         </div>
+        <p class="card-text"><small class="text-muted"></small></p>
       </div>
```
Comment (jefflaub12) [inline views/shop/pricing.hbs:20]: Is this an extra /div?

File: views/shop/pricing.hbs [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
+      {{/each}}
+  </div>j
 </div>
-{{/each}}
+  {{/each}}
+
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+</body>3
```
Comment (jefflaub12) [inline views/shop/pricing.hbs:30]: What is 3?

File: views/shop/pricing.hbs (hunk 1 above)
Comment (brucekirk735) [reply views/shop/pricing.hbs:23]: j is for jerk

**Pull request closed without merging**